heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -87,6 +104,13 @@ function process_message()
         msg.Fields.clientId         = parsed.clientID
     elseif parsed.version then
         -- New-style telemetry, see http://mzl.la/1zobT1S
+        if parsed.type == "main" then
+            msg.Payload = payload -- keep the gzipped payload since we are moving most of the content into fields
```

Comment:
Do we want to keep it? The original payload (gzipped in most cases), the full JSON, the remaining JSON after the object extraction?